iree/turbine/kernel/wave/runtime/CMakeLists.txt

```diff
@@ -0,0 +1,66 @@
+cmake_minimum_required(VERSION 3.15...3.27)
+project(wave_runtime)
+
+if (CMAKE_VERSION VERSION_LESS 3.18)
+  set(DEV_MODULE Development)
+else()
+  set(DEV_MODULE Development.Module)
+endif()
+
+find_package(Python 3.10 COMPONENTS Interpreter ${DEV_MODULE} REQUIRED)
```

Comment:
Silly Q, if we use 3.10 here, and assume I have 3.10 and 3.12 installed on my local machine but my venv is running at 3.12 , would this still work? Additionally, what if my machine do not have a 3.10 but only a 3.12?

Not needed in this PR itself but perhaps need to add a TODO here to make it work with different python versions..

Reply:
So the 3.10 here represents a minimum version. But good question since I ran into something similar on the ci. The find package here is being guided by the way we invoke cmake. So if you see we pass in the path of the python executable that is running the wave code and this way we make sure that it doesnt pick up some other system python installed.